src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -0,0 +1,330 @@
+from copy import deepcopy
+from typing import Literal, Optional
+
+import numpy as np
+from pydantic import FilePath, validate_call
+from pynwb import NWBFile
+from pynwb.image import ImageSeries
+
+from .video_utils import VideoCaptureContext
+from ....basedatainterface import BaseDataInterface
+from ....tools import get_package
+from ....tools.nwb_helpers import get_module
+from ....utils import dict_deep_update, get_base_schema, get_schema_from_hdmf_class
+
+
+class ExternalVideoInterface(BaseDataInterface):
+    """Data interface for writing videos as ImageSeries."""
```

Comment:
Something something external series. 

The docstring should reference the other interface and where do they stand with respect to each other I think.